src/extras/MovieClip.js

```diff
@@ -170,8 +203,16 @@ MovieClip.prototype.gotoAndStop = function (frameNumber)
 
     this._currentTime = frameNumber;
 
-    var round = Math.floor(this._currentTime);
-    this._texture = this._textures[round % this._textures.length];
+    if (this._durations !== null)
```

Comment:
You need to update the _lag property because you change currentTime. In case you set the frame to 1.5 the lag property is set so it's the half of the duration of frame 1